magento/magento2
# Issue 5316

### Description

Fix HTML minification problem, when there is a php a comment and no space at the php end tag

### Fixed Issues (if relevant)

1. magento/magento2#5316: [2.1.0] HTML minification problem with php tag with a comment and no space at the end

### Manual testing scenarios
1. Create a theme and a template file which contains this:

```
<?php //if ($showDescription):?>
    <div class="product description product-item-description">
        <?php /* @escapeNotVerified */ echo $_helper->productAttribute($_product, $_product->getShortDescription(), 'short_description') ?>
    </div>
<?php //endif; ?>
```

2. Enable HTML minification in the backend
3. Re-deploy static content and view_preprocessed in order to generate minified templates
4. Check the front page, where template is used.

### Contribution checklist
